lib/vanagon/platform/windows.rb

```diff
@@ -21,9 +21,9 @@ def generate_package(project)
       # @param project [Vanagon::Project] project to name
       # @return [String] name of the windows package for this project
       def package_name(project)
-        # If nothing is passed in as platform type, default to a nuget package
-        # We should default to an MSI once that code has been implimented
```

Comment:
:+1: with the new vanagon updates, we're setting the default to MSI. So if people want to build nuget packages, or build packages while we are still working on the MSI stuff, they have to explicitely set package_type to nuget.